Ensure to update NPM to the latest version compatible with the installed Node version (#21)
OpenTermsArchive/deployment: CHANGELOG.md

### Changed

- Make the publication process compatible with the protected branch

### Fixed

> - Ensure to update NPM to the latest version compatible with the version of Node installed

Reviewer: ```suggestion
- Ensure NPM version is compatible with NodeJS version
```
